### PR TITLE
Login

### PR DESCRIPTION
**Pull request by mcallegari10**
## Description

Add login view

## Screenshot

image

### CODE REVIEW

File: src/App/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const user = {
+      email: event.target.user.value,
+      password: event.target.password.value
+    };
+    login(user).then((response) => {
+      setLoginInfo(response.data);
```
Comment (pabloferro) [inline src/App/components/Login/index.js:20]: Why don't you do this on the service?

File: src/auth.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export function setLoginInfo(data) {
+  localStorage.setItem('renew_id', data.renew_id)
+  localStorage.setItem('access_token', data.access_token)
```
Comment (pabloferro) [inline src/auth.js:3]: Shouldn't you update api config here?

File: src/App/components/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      history.push('/dashboard');
```
Comment (pabloferro) [inline src/App/components/Login/index.js:21]: Why don't you use react-router to change the location?

**Review by pabloferro — CHANGES_REQUESTED** [review]

File: src/App/components/Login/index.js (hunk 3 above)
Comment (mcallegari10) [reply src/App/components/Login/index.js:21]: This is the way you change the route with React Router 4 :disappointed:

File: src/auth.js (hunk 2 above)
Comment (mcallegari10) [reply src/auth.js:3]: What do you mean by update api config? :thinking:

Comment (pabloferro) [reply src/auth.js:3]: The Authorization header.

**Pull request closed without merging**